slack.ts

```diff
@@ -0,0 +1,54 @@
+/** Error20:
+    - line 21: don't know how to declare call signature in tsc*/
+
+import * as querystring from "querystring";
+import fetch from "node-fetch";
+
+const tkn = process.env.SLACK_TOKEN;
+const domain = process.env.DOMAIN;
```

Comment:
probably best to rename to `SLACK_DOMAIN` in case we want to use the word `DOMAIN` for something else.